conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -38,18 +41,17 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-
+conda install --yes conda-build==1.18.2
```

Comment:
Sounds good to me. It would be great if every workaround is commented with links to related issues/PRs right in the sources, so it is obvious what was wrong with it and when it is safe to remove the workaround. I am maintaining our company's recipes and I know that almost every release of Conda/Conda-Build breaks something somewhere in an unexpected way, this is why I asked what is wrong with all newer Conda-Build releases after 1.18.2.